keyboards/dotmod/dymium65/info.json

```diff
@@ -0,0 +1,137 @@
+{
+    "keyboard_name": "DYMIUM65",
+    "manufacturer": "RDR",
+    "url": "https://github.com/LiuLiuQMK",
+    "maintainer": "LiWenLiu",
+    "usb": {
+        "vid": "0x320F",
```

Comment:
This vendor ID is already in use, and should be changed.

https://www.the-sz.com/products/usbid/index.php?v=0x320F&p=&n=

Above 0x4000 is good, and the further past that, the better.

Reply:
Thanks for your reminder!